mlrun/config.py

```diff
@@ -1038,6 +1041,19 @@ def verify_security_context_enrichment_mode_is_allowed(self):
                 f"is not allowed for iguazio version: {igz_version} < 3.5.1"
             )
 
+    @staticmethod
+    def validate_alert_activation_retention():
```

Comment:
As mentioned in previous comment - I think this needs to be `validate_object_retentions`, and internally iterate over all members of that config (currently only alert activations).